third_party/harfbuzz.BUILD

```diff
@@ -63,7 +63,29 @@ cc_library(
         "src/hb-subset-repacker.h",
         "src/hb-unicode.h",
         "src/hb-version.h",
-    ],
+        "src/OT/Layout/GSUB/AlternateSet.hh",
```

Comment:
As these are private headers these would be listed in the 'srcs' attribute. You can add a glob pattern for them.